src/Network/Network/PublicIpPrefix/NewAzurePublicIpPrefixCommand.cs

```diff
@@ -96,6 +96,12 @@ public class NewAzurePublicIpPrefixCommand : PublicIpPrefixBaseCmdlet
             ValueFromPipelineByPropertyName = true)]
         public string[] Zone { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The CustomIpPrefix that this PublicIpPrefix will be associated with")]
+        public PSCustomIpPrefix CustomIpPrefix { get; set; }
```

Comment:
Help doc of this cmdlet should be updated.
Would be great to add an example

Reply:
updated help doc - I think we want to hold off on the example; this feature is still in private preview so I don't want to make it too obvious for the world to see and use this feature. Furthermore, there's a chance that things change so I kind of want to keep this feature somewhat hidden for the time being. Once we go into public preview/GA, then we will solidify all the documentation.